README.md

```diff
@@ -35,6 +35,12 @@ the review process quicker and more streamlined.
 
 ## Install
 
+To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
+The oldest Node.js version we support is 4.2.0.
```

Comment:
See https://nodejs.org/en/download/releases/ and https://github.com/nodejs/node/blob/v4.2.0/CHANGELOG.md

> Version 4.2.0 'Argon'
> The first Node.js LTS release! See https://github.com/nodejs/LTS/ for details of the LTS process.